test/e2e/antctl_test.go

```diff
@@ -33,6 +33,26 @@ type cmdAndReturnCode struct {
 	expectedReturnCode int
 }
 
+// TestAllAntctl is the top-level test which contains all subtests for
+// Antctl related test cases as they can share setup, teardown.
+func TestAllAntctl(t *testing.T) {
```

Comment:
Can we just say "TestAntctl"? I believe all subtests have a suffix describing its focus. Prepending "All" to all tests seems redundant.

Reply:
sure